libs/components/packages/src/schematics/migrations/update-7/add-compat-stylesheets.spec.ts

```diff
@@ -16,18 +16,18 @@ describe('Migrations > Add compat stylesheets', () => {
  * - Delete this file after all blocks have been addressed.
  * - Remove each occurrence of this file in your project's
  *   angular.json file.
-*******************************************************************************/
+ *******************************************************************************/
```

Comment:
Was the addition of this space intentional?

Reply:
Yep. It's not super important, but it converts the comment block format from:
```
/**
 *
*******/
```

To:
```
/**
 *
 *******/
```
Which is more in line with how JSDocs creates their comment blocks. I can revert if you'd like.